jobsdb/jobsdb.go

```diff
@@ -2070,9 +2066,7 @@ func (jd *HandleT) storeJob(ctx context.Context, tx *Tx, ds dataSetT, job *JobT)
 	_, err = stmt.ExecContext(ctx, job.UUID, job.UserID, job.CustomVal, string(job.Parameters), string(job.EventPayload), job.WorkspaceId)
 	if err == nil {
 		tx.AddSuccessListener(func() {
-			// Empty customValFilters means we want to clear for all
-			jd.markClearEmptyResult(ds, allWorkspaces, []string{}, []string{}, nil, hasJobs, nil)
-			jd.markClearEmptyResult(ds, job.WorkspaceId, []string{}, []string{}, nil, hasJobs, nil)
```

Comment:
**Note:** previously we were invalidating the whole cache if we were storing singular jobs